portal/code/cx-portal/src/utils/dataMapper.ts

```diff
@@ -0,0 +1,47 @@
+import {
```

Comment:
for this kind of logic we should provide Unit tests - though it's a very simple one here

Reply:
Would be nice, I noted it and I will provide in another PR